src/werkzeug/posixemulation.py

```diff
@@ -73,10 +73,10 @@ def _rename_atomic(src, dst):
 
     except Exception:
 
-        def _rename(src, dst):
+        def _rename(_src, _dst):
```

Comment:
`posixemulation` was removed, but regardless, this no longer matches the signature of the other version of these methods.